pdns/dnsdist-lua.cc

```diff
@@ -855,15 +855,28 @@ static void setupLuaConfig(LuaContext& luaCtx, bool client, bool configCheck)
         return;
       }
 
+      if (password || apiKey || customHeaders || acl) {
+	warnlog("Passing additional parameters to 'webserver()' is deprecated, please use 'setWebserverConfig()' instead.");
```

Comment:
nit: needs an indent

Reply:
As it turns out we still have some stranded tabs here and there, fixed.